src/commands/temp.js

```diff
@@ -0,0 +1,61 @@
+import { exampleCommand, beep } from '../utilities/string'
+import { getRandomInt } from '../utilities/number'
+
+// site for get TextToSpeech
+// https://soundoftext.com/#learn
+
+const voiceUrls = {
```

Comment:
seperate file name and url, then add a function to get the full url.
you can use same variable for `voiceUrls` and `voiceNumberUrls`
``` js
const VOICE_URL = 'https://soundoftext.nyc3.digitaloceanspaces.com'
const VOICE_FILE_NAME = {
  yourTemp: '...',
  thirtySixDot: '...',
  1: '...',
  2: '...',
  // ...
}
const getVoiceLine = (name) => `${VOICE_URL}/${VOICE_FILE_NAME[name]}`
```